app/src/Processors/BuildsProcessor.php

```diff
@@ -156,6 +157,11 @@ public function process(bool $refetch): array
                 }
             }
 
+            $pmPrevMinBrn = '';
+            if ($pmPatBrn && preg_match('/\.[1-9][0-9]*$/', $pmPatBrn)) {
+                $pmPrevMinBrn = $pmPatBrn - 0.1;
+            }
```

Comment:
Can you please add a comment above this saying why we're doing this? It's not clear what the purpose of this is.

Reply:
Updated